lib/Onyx.js

```diff
@@ -150,6 +150,45 @@ function addAllSafeEvictionKeysToRecentlyAccessedList() {
         });
 }
 
+/**
+ * When a collection of keys change, search for any callbacks matching the collection key and trigger those callbacks
+ *
+ * @param {String} parentKey
+ * @param {Object} collection
+ */
+function keysChanged(parentKey, collection) {
```

Comment:
Rather than call this a `parentKey` could we call it a `collectionKey` ? I think it is more in line with the terminology we are using in Onyx so far.